website/lib/changelog/generate-component-changelog-entries.mjs

```diff
@@ -180,7 +131,7 @@ const updateComponentVersionHistory = (componentChangelogEntries, version) => {
 
 const updateComponentFrontMatter = (componentChangelogEntries, version) => {
   Object.keys(componentChangelogEntries).forEach((componentName) => {
-    const indexPath = `${allComponentsPath[componentName]}/index.md`;
+    const indexPath = `${allComponentPaths[componentName]}/index.md`;
```

Comment:
[issue/blocker] same as above `allComponentPaths` is not an object now, but an array

Reply:
note: I am not even sure you need `allComponentPaths` at all anymore: you have the component path directly from the regex/`START` itself, no?